README.md

```diff
@@ -270,7 +268,8 @@ ElasticsearchEvolution.configure()
 ## 6 changelog
 
 ### v0.3.0-SNAPSHOT
-
+-   version upgrade elasticsearch-rest-high-level-client to 7.5.0 (Es version < 7.5.0 are no longer supported)
```

Comment:
version upgrade elasticsearch-rest-high-level-client to 7.5.2 (Es version < 7.5.0 are no longer supported)